src/hooks/useGrantsByUser.ts

```diff
@@ -0,0 +1,12 @@
+import useAsyncMemo from 'decentraland-gatsby/dist/hooks/useAsyncMemo'
+
+import { Governance } from '../clients/Governance'
+import { GrantsResponse } from '../entities/Proposal/types'
+
+export default function useGrantsByUser(address: string, coauthoring?: boolean) {
+  return useAsyncMemo(
+    async () => await Governance.get().getGrantsByUser(address, coauthoring),
```

Comment:
In what case `coauthoring` is false?

Reply:
For the time being, never. I left it ready in case we need that option in the future.